server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -0,0 +1,392 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Hasura.Eventing.ScheduledTrigger
```

Comment:
This module could also really benefit from some module-level documentation (or it should at least point to documentation somewhere else).